server/viame_server/serializers/meva.py

```diff
@@ -0,0 +1,242 @@
+    for activity_packet in yml:
+        if kpf.ACTIVITY in activity_packet:
+            activity = _deserialize_activity(activity_packet, actor_map)
+            activity_map[activity.activity_id] = activity
+
+def _deserialize_activity(activity_packet, actor_map):
+    """
+    returns activity instanceActor(
+    """
+    activity = activity_packet[kpf.ACTIVITY]
+    timespans = activity[kpf.TIMESPANS]
+    frame_timespan = kpf._deserialize_frame_timespan(timespans)
+    actors = activity[kpf.ACTORS]
+    activity_type_obj = activity[kpf.ACTIVITY_TYPE]
+    activity_type_keys = list(activity_type_obj.keys())
+    if len(activity_type_keys) != 1:
+        raise BoilerError(f'{activity_type_obj} should only have 1 key')
+    activity_type = activity_type_keys[0]
+    confidence = activity_type_obj[activity_type]
+    status = None
+    if kpf.STATUS in activity:
+        # status = ActivityPipelineStatuses(activity[kpf.STATUS])
+        status = activity[kpf.STATUS]
+
+    return models.Activity(  # type: ignore
+        activity_id=activity[kpf.ACTIVITY_ID],
+        activity_type=activity_type,
+        begin=frame_timespan[0],
+        end=frame_timespan[1],
+        status=status,
+        actors=[_deserialize_actor(a, actor_map, activity[kpf.ACTIVITY_ID], activity_type, confidence, status) for a in actors],
+    )
+
+def _deserialize_actor(actor, actor_map, activity_id, activity_type, confidence, status):
+    if kpf.ACTOR_ID not in actor:
+        raise BoilerError(f'actor {actor} missing {kpf.ACTOR_ID}')
+    if kpf.TIMESPANS not in actor:
+        raise BoilerError(f'actor {actor} missing {kpf.TIMESPANS}')
+    actor_id = actor[kpf.ACTOR_ID]
+    timespans = actor[kpf.TIMESPANS]
+    frame_timespan = kpf._deserialize_frame_timespan(timespans)
+    if actor_id in actor_map:
+        actor_map[actor_id].begin = frame_timespan[0]
+        actor_map[actor_id].end = frame_timespan[1]
+        actor_map[actor_id].activity_id = activity_id
+        actor_map[actor_id].activity = activity_type
+        actor_map[actor_id].src_status = status
+        actor_map[actor_id].activity_con = confidence
+    else:
+        actor_map[actor_id] = Actor(  # type: ignore
+            clip_id=actor_id, begin=frame_timespan[0], end=frame_timespan[1],
+            activity_id=activity_id, activity=activity_type, src_status=status, confidence=confidence
+        )
+    return actor_map[actor_id]
+
+
+def serialize_types(actor_map: Dict[int, models.Actor]):
+    kpf.serialize_types(actor_map)
+
+def serialize_geom(actor_map: Dict[int, models.Actor]):
+    kpf.serialize_geom(actor_map)
+
+def serialize_activities(activity_map: Dict[int, models.Activity], actor_map: Dict[int, models.Actor]):
+    kpf.serialize_activities(activity_map, actor_map)
+
+def serialize_to_files(
+    output_basename: str, activity_list: models.ActivityList, keyframes_only=False,
+):
+    kpf.serialize_to_files(output_basename, activity_list, keyframes_only)
```

Comment:
Can probably remove these wrapper functions,  if we want serialization, can use the boiler methods directly.